lib/loadGltfUris.js

```diff
@@ -94,7 +98,12 @@ function loadURI(gltf, basePath, name, callback) {
     }
 }
 
+//Return the extension of the data uri
  function getImageDataUriFormat(uri) {
-    //Return the extension of the data uri
-    return '.' + uri.match('\/([^)]+)\;')[1];
+    var extension = uri.match('\/([^)]+)\;');
+    if (extension === null) {
```

Comment:
Is assuming `.png` the right thing to do here?  Does this case ever happen for valid data URIs?  Would it be better to throw an exception?